spec/reciter_spec.rb

```diff
@@ -3,10 +3,37 @@
 describe Reciter do
 
   let(:rhyme_path) { 'spec/fixtures/the_house_that_jack_built.txt' }
+  let(:rhyme_phrases_path) { 'spec/fixtures/the_house_that_jack_built_phrases.txt' }
+
+  describe '#recite_rhyme' do
+
+    it 'returns a string for the rhyme' do
+      rhyme = File.open(rhyme_path).read
+      expect(subject.recite_rhyme).to eq(rhyme)
+    end
+
+  end
+
+  describe '#recite_random_rhyme' do
+
+    it 'returns a string for the rhyme with each element in random order' do
+      phrases = []
+      File.open(rhyme_phrases_path).each_line do |line|
+        phrases.push(line)
+      end
+
+      random_indices = subject.generate_random_indices
```

Comment:
A simpler test that doesn't rely on the code under test itself would be to check the exact string value for a couple different random index arrays.